public/modules/groups/controllers/services.groups.client.controller.js

```diff
@@ -4,9 +4,10 @@ angular.module('groups').controller('ServicesGroupsController', ['$scope', '$sta
     function ($scope, $stateParams, $location, Authentication, Groups, Services, Daemons, GroupsServices) {
         $scope.authentication = Authentication;
 
-        $scope.patternName = /^[a-zA-Z0-9_\-/]{1,200}$/;
+        $scope.patternName = /^[a-zA-Z0-9_\.\-/]{1,200}$/;
```

Comment:
❓ Why do we allow dot in container name ? Maybe `MyContainer.local` ? 
 Are you sure there is no impact ?

Reply:
🤔

Reply:
We add a dot to have a container name like that `mycontainername.local` to just have to add `.local` in no_proxy and not every containers name. You can check there https://github.com/moby/moby/blob/be97c66708c24727836a22247319ff2943d91a03/daemon/names/names.go